api/src/main/resources/messages.properties

```diff
@@ -2944,6 +2944,11 @@ install.advanced.other.update=Do you want updates to the database to be automati
 install.advanced.admin=Step 4 of 5
 install.advanced.admin.password.info=Currently the default administrator user/pass is <strong>admin:test</strong>. Please specify a new password for that user
 install.advanced.admin.password.retype=Retype password
+install.advanced.admin.password.constraints=Your Password should follow the following constraints
+install.advanced.admin.password.constraints.min.length=Contains minimum 8 characters
+install.advanced.admin.password.constraints.cases=Contains both upper and lowercase letters
+install.advanced.admin.password.constraints.numbers=Contains numbers
```

Comment:
Contains numbers appears to be a vague statement. Shouldn't it be like "Contains at least one number".?

Reply:
Yes that's sounds good. I'll change that. Thanks @Am-Coder!